docs/_posts/examples/v1.0.0/0100-01-01-markercluster-custom-marker-icons.html

```diff
@@ -16,12 +16,12 @@
 <script>
 // We add the backing styleLayer and use the featureLayer for its data.
 var map = L.mapbox.map('map')
-    .setView([40.73, -74.011], 13)
+    .setView([39.656, -97.273], 5)
     .addLayer(L.mapbox.styleLayer('mapbox://styles/mapbox/streets-v11'));
 
 // Since featureLayer is an asynchronous method, we use the `.on('ready'`
 // call to only use its marker data once we know it is actually loaded.
-L.mapbox.featureLayer('examples.map-h61e8o8e').on('ready', function(e) {
+L.mapbox.featureLayer('https://docs.mapbox.com/mapbox-gl-js/assets/earthquakes.geojson').on('ready', function(e) {
```

Comment:
I'm seeing some broken icons on this map (no errors logged to console), could it be related to changing the datasource?

![image](https://user-images.githubusercontent.com/2180540/101367894-8691dc80-3874-11eb-837b-133085af357e.png)

Reply:
Yep, turns out the style relies on some simple style spec properties that weren't available in that data. I've updated to a better data source in b6988728.